src/common/pf_lldp.c

```diff
@@ -1881,27 +1914,41 @@ void pf_lldp_update_peer (
       return;
    }
 
-   error = pf_lldp_generate_alias_name (
-      lldp_peer_info->port_id.string,
-      lldp_peer_info->chassis_id.string,
-      new_alias,
-      sizeof (new_alias));
-   if (!error && (strcmp (new_alias, net->cmina_current_dcp_ase.alias_name) != 0))
-   {
-      LOG_INFO (
-         PF_LLDP_LOG,
-         "LLDP(%d): Updating alias name: %s -> %s\n",
-         __LINE__,
-         net->cmina_current_dcp_ase.alias_name,
-         new_alias);
+   LOG_INFO (
+      PF_LLDP_LOG,
+      "LLDP(%d): Port %d, peer info changed\n",
+      __LINE__,
+      loc_port_num);
 
-      strncpy (
-         net->cmina_current_dcp_ase.alias_name,
-         new_alias,
-         sizeof (net->cmina_current_dcp_ase.alias_name));
+   LOG_INFO (
```

Comment:
Remember to show the old peer info only if `p_port_data->lldp.is_peer_info_received`

Reply:
Done